### PR TITLE
Fixes #941: Added a wiki for Sharing data across pages

### PR DESCRIPTION
**Pull request by deepakchethan**
Fixes https://github.com/appsmithorg/appsmith/issues/941

### CODE REVIEW

**Comment by deepakchethan** [comment]
@Nikhil-Nandagopal, @hetunandu Please see if this is fine

**Review by hetunandu — CHANGES_REQUESTED** [review]
Everything looks really good except for the local storage example gif. A minor nitpick but the gif should have a user writing a name like "Deepak" and the next page should have a message like, "Hello Deepak, welcome to Appsmith"

File: SUMMARY.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 * [Connecting UI & Logic](core-concepts/connecting-ui-and-logic/README.md)
   * [Working with JS libraries](core-concepts/connecting-ui-and-logic/working-with-js-libraries.md)
   * [Internal Functions](core-concepts/connecting-ui-and-logic/internal-functions.md)
+  * [Data Sharing Across Pages](core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md)
```
Comment (Nikhil-Nandagopal) [inline SUMMARY.md:25]: Can we change the text to 
Sharing Data Across Pages

File: core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+description: >-
+  The Appsmith platform allows us to share data across pages either via query params or via local storage.
```
Comment (Nikhil-Nandagopal) [inline core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md:3]: replace "either via..." with "using URL query params & the browser local storage"

File: core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+# Data Sharing Across Pages
+
+## Sharing data via query params 
+
+We can pass query params to a given page by adding the query params object to `NavigateTo` Action. To do this click on the JS mode for Action and update it as follows.
```
Comment (Nikhil-Nandagopal) [inline core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md:10]: Can you change the sentences to passive voice everywhere on the page? 
i.e "Query params can be passed by adding..."
instead "of we can pass ...."

File: core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+```
+{{navigateTo("PageName", {"param": "value"})}} 
+```
+These can be consumed on the destination page with `appsmith.URL.queryParams.param`.
+
+![Click to expand](../../.gitbook/assets/queryParams.gif)
+
+## Sharing data via local storage 
+
+We can store the key value pairs within the local storage with the help of `StoreValue` function. To do so pick the StoreValue action and provide the desired key and values.
```
Comment (Nikhil-Nandagopal) [inline core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md:20]: Please link to the navigate and storeValue function docs as well

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
@deepakchethan thank you for the contribution! I've added a few comments. We can merge this once they are resolved

File: SUMMARY.md (hunk 1 above)
Comment (deepakchethan) [reply SUMMARY.md:25]: Done

File: core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md (hunk 2 above)
Comment (deepakchethan) [reply core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md:3]: Done

File: core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md (hunk 3 above)
Comment (deepakchethan) [reply core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md:10]: Done

File: core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md (hunk 4 above)
Comment (deepakchethan) [reply core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md:20]: I have linked navigateTo, but we still don't have storeValue function doc as of now.

**Review by hetunandu — APPROVED** [review]
LGTM

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Comment by Nikhil-Nandagopal** [comment]
@deepakchethan thank you for your contribution!

**Pull request merged**